packages/plugin-css/test/border/tokens.json

```diff
@@ -69,6 +69,9 @@
         "width": "1px",
         "style": "solid"
       }
+    },
+    "default": {
+      "$value": "{border.solid}"
```

Comment:
Also tested `border` and `gradient` tokens to make sure it wasn’t happening with other composite types